modules/local/selfcomp_splitfasta.nf

```diff
@@ -1,6 +1,6 @@
 process SELFCOMP_SPLITFASTA {
     tag "$meta.id"
-    label 'process_single'
+    label 'process_medium'
```

Comment:
As far as I can see, this module (and `modules/local/selfcomp_alignmentblocks.nf`, `modules/local/selfcomp_mummer2bed.nf` and `modules/local/selfcomp_mapids.nf`) are single-threaded. `process_medium` increases both the memory and the CPU count, and will result in wasting CPU slots on the farm. If you need the equivalent of `process_medium` in terms of memory, you can either create your own label and define it in `conf/base.config` with `withLabel:`, but I fear `nf-core lint` may complain it doesn't recognise the label. Alternatively, remove the labels in those four modules, and go with something like `withName: '.*:.*:SELFCOMP:(SELFCOMP_ALIGNMENTBLOCKS|SELFCOMP_MAPIDS|SELFCOMP_MUMMER2BED|SELFCOMP_SPLITFASTA)'`

Reply:
I'll add the last line to the modules.config and remove labels from the local modules.